website/blog/2020-06-19-getting-started-and-aria.md

```diff
@@ -0,0 +1,213 @@
+
+Let's take a look at getting a Docker image together for Presto (though they already exist on Dockerhub, e.g. [ahanaio/prestodb-sandbox](https://hub.docker.com/r/ahanaio/prestodb-sandbox)).  We can see below how relatively easy it is to get Presto up and running.  We download Presto, copy some configuration files in a local `etc` directory into the image, and specify an entry point to run the server.
+
+```Dockerfile
+FROM openjdk:8-jre
+
+# Pick our Presto Version and the URL to download
+ARG PRESTO_VERSION=0.235.1
+ARG PRESTO_BIN=https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
+
+# Update the base image OS and install wget and python
+RUN apt-get update
+RUN apt-get install -y wget python
+
+# Download Presto and unpack it to /opt/presto
+RUN wget --quiet ${PRESTO_BIN}
+RUN mkdir -p /opt
+RUN tar -xf presto-server-${PRESTO_VERSION}.tar.gz -C /opt
+RUN rm presto-server-${PRESTO_VERSION}.tar.gz
+RUN ln -s /opt/presto-server-${PRESTO_VERSION} /opt/presto
+
+# Copy configuration files on the host into the image
+COPY etc /opt/presto/etc
+
+# Download the Presto CLI and put it in the image
+RUN wget --quiet https://repo1.maven.org/maven2/com/facebook/presto/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar
+RUN mv presto-cli-${PRESTO_VERSION}-executable.jar /usr/local/bin/presto
+RUN chmod +x /usr/local/bin/presto
+
+# Specify the entrypoint to start
+ENTRYPOINT /opt/presto/bin/launcher run
+```
+
+There are four files in the `etc` folder to configure Presto, along with two catalogs. Details of them can be found [here](https://prestodb.io/docs/current/installation/deployment.html).  They are:
+
+```bash
+etc/
+├── catalog
+│   ├── hive.properties  # Defines configuration properties for our Hive Metastore
+│   └── tpch.properties  # Enables the TPCH connector to generate data
+├── config.properties 	 # Presto instance configuration properties
+├── jvm.config           # JVM configuration for the process
+├── log.properties       # Logging configuration
+└── node.properties      # Node-specific configuration properties
+```
+
+
+The four files directly under `etc` are documented in the above link.  Let's look at the details of the two catalogs: `hive` and `tpch`. The former will be used to read data, and the latter will be used to generate data into our Hive warehouse to run queries.  Setting up Hive is outside the scope of this document, but will be covered in a future post.  In short, you need a catalog service in order to use the Hive connector, such as a Hive Metastore or AWS Glue.  The catalog service contains all of the table definitions used by Presto, and data is typically stored in HDFS or S3.  We've set up a Dockerized Hive Metastore and single-node HDFS instance.
```

Comment:
This paragraph would be a good start for a subsection about configuring a connector with TPC-H as an example